### PR TITLE
Consolidate local playbooks

### PR DESCRIPTION
**Pull request by cfchase**

### CODE REVIEW

**Comment by cfchase** [comment]
@jwmatthews what do you think about consolidating our local playbooks and using variables instead?

File: ansible/roles/ansible_service_broker_setup/tasks/main.yml [hunk 1]
```diff
@@ -39,7 +39,6 @@
       -p DOCKERHUB_USER={{ dockerhub_user_name }}
       -p LAUNCH_APB_ON_BIND={{ broker_launch_apb_on_bind }}
       -p OUTPUT_REQUEST={{ broker_output_request }}
-      -p RECOVERY={{ broker_recovery }}
```
Comment (jwmatthews) [inline ansible/roles/ansible_service_broker_setup/tasks/main.yml]: I think we will want to add this back now that the recovery pr has merged

Comment (cfchase) [inline ansible/roles/ansible_service_broker_setup/tasks/main.yml]: thanks for catching that, that was unintentional

File: ansible/roles/ansible_service_broker_setup/tasks/main.yml [hunk 2]
```diff
@@ -78,7 +78,7 @@
       url: http://{{ ansible_service_broker_route }}/v2/bootstrap
       method: POST
     register: response
-    until: response|success and response.json and response.json.spec_count
+    until: response|success and response.json and response.json.spec_count is defined
```
Comment (cfchase) [inline ansible/roles/ansible_service_broker_setup/tasks/main.yml]: This change keeps catasb from failing whenever someone doesn't have any apbs yet.

Comment (jwmatthews) [inline ansible/roles/ansible_service_broker_setup/tasks/main.yml]: +1

**Comment by jwmatthews** [comment]
Would you also update the README's to coincide, assume we need some directions to create my_vars.yml a level up, also some mention of the other .yml being shared in local directory.

File: ansible/roles/ansible_service_broker_setup/tasks/main.yml (hunk 2 above)
Comment (dymurray) [inline ansible/roles/ansible_service_broker_setup/tasks/main.yml]: +1

File: local/config/my_vars.yml.example [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+# Override any and all Ansible variables when running catasb setup scripts.
+
+# For convenience you can skip being prompted for these vars by storing them here.
+dockerhub_user_name: example_user
+dockerhub_org_name: example_org
+
+# Omit this unless you want to store your password in plain text to skip prompts.
+# dockerhub_user_password: example_password
```
Comment (dymurray) [inline local/config/my_vars.yml.example]: Good idea to have a special note about this

**Pull request merged**